_examples/iface/iface.go

```diff
@@ -27,3 +30,13 @@ func CallIface(v Iface) {
 	v.F()
 	cpkg.Printf("iface.CallIface... [DONE]\n")
 }
+
+// by default, interface{} is converted to string (most universal type)
+func IfaceString(str interface{}) {
+	cpkg.Printf("iface as string: %v\n", str)
+}
+
+//gopy:interface=handle this magic directive says, treat it as a handle
```

Comment:
could/shouldn't the directive be put on its own line (to better follow all the other `//go:something` directives) ?